modules/cnf-installing-the-n3000-operator.adoc

```diff
@@ -0,0 +1,162 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="installing-the-openness-operator-intel-fpga-pac-n3000-operator_{context}"]
```

Comment:
Please set the ID to match the file name according to the second item in the [module file metadata](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata) section of the doc guidelines.

Reply:
Please set the ID according to the guidelines in the preceding comment.

Reply:
Please set the ID as follows:

```
[id="installing-the-n3000-operator_{context}"]
```

If the ID cannot be set to that value--to meet the doc guidelines--please indicate the reason in a comment so that a subsequent reviewer does not request the same change.